config/default.yml

```diff
@@ -3724,6 +3724,8 @@ Style/HashExcept:
                  that can be replaced with `Hash#except` method.
   Enabled: pending
   VersionAdded: '1.7'
+  VersionChanged: '<<next>>'
+  ActiveSupportEnabled: false
```

Comment:
I've added `ActiveSupportEnabled` option to switch checking methods extended by ActiveSupport.
The option name might have better alternatives 🤔

Reply:
@nobuyo I like @Darhazer's idea to have some shared config option for this (perhaps a global config option?), so all the impacted cop can check for the same thing. The name of the option is fine, although if we want to be more specific I guess it should `ActiveSupportExtensionsEnabled` or however they call them these days. I'm fine with the current name as well.

@rubocop/rubocop-core any thoughts on this?

Reply:
Sorry I was misunderstanding. I'll start over with a new way to create a new mixin.